recipes/annotsv/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -euo pipefail
+
+make PREFIX="${PREFIX}" install
+make PREFIX="${PREFIX}" install-human-annotation
```

Comment:
I'm testing this locally, and this seems to be downloading quite large amounts of data. I'm going to go out on a limb and suggest that this is not going to fly (it would get baked into containers etc). Is there a reason that can't be done at run time?

Reply:
All it's doing is downloading and uncompressing a (rather large) tar, I think don't think these should be in the recipe.

The first `make` command can also go in the meta.yaml under build/script if you'd like to save a file.

Reply:
There is a `annotationsDir` flag we can use. I'll remove the annotation files from the recipe now

Reply:
I mean you don't actually need the build.sh if the build command is simple, you can embed it in the meta.yaml - see e.g. https://github.com/bioconda/bioconda-recipes/blob/master/recipes/mdasim/meta.yaml

Reply:
OOOOOH interesting! Thanks!